api/routes/tasks.py

```diff
@@ -1,25 +1,82 @@
+import datetime
+
 from app import app
 from models import db
 from models.task import Task
+from models.slot import Slot
 from util.response import json_response
 from util.user_id import with_user_id
 from flask import request, Response
 from sqlalchemy.dialects import postgresql
 
 
-@app.route("/api/tasks", methods=["GET"])
+@app.route("/api/task/get-all", methods=["GET"])
 @with_user_id
 def get_tasks(user_id):
     query = db.select(Task).filter(Task.user_id == user_id).order_by(Task.created)
     tasks = db.session.execute(query).scalars().all()
-    return json_response([{"name": t.title, "id": t.id} for t in tasks])
+    return json_response(
+        [
+            {
+                "name": t.title,
+                "id": t.id,
+                "description": t.description,
+                "complete": t.complete,
```

Comment:
once you change `t.complete` to point to the slot, this needs to become a non-null check

Reply:
Yep